### ISSUE TITLE
error load image on IOS

### ISSUE THREAD

**Issue by DoVanBachTlu** [DylanVann/react-native-fast-image#1058]
react-native-fast-image 8.6.3. 
react-native: 0.70.13
if i exit the app and re-enter it works
The problem here is that I was using it normally and then it suddenly got an error even though I didn't update anything.

**Comment by Inn116mac** [comment]
is there any update?

**Comment by DoVanBachTlu** [comment]
no. i switched to https://github.com/candlefinance/faster-image. base performance is almost same as fast image

> is there any update?
